### ISSUE TITLE
Limit Feed Per Channel | Lastest 3 Videos Only

### ISSUE THREAD

**Issue by mrlen76** [FreeTubeApp/FreeTube#1245]
Some channels tend to upload very frequently. So when you look st your subs, you just see a whole page full of their videos. It would be nice if we could have a setting where we can just display the last 3 videos of each channel -- and if we want to see more, we can just go to their actual channel/profile. This functionality would be AWESOME!

**Comment by Svallinn** [comment]
We are already have a feature that provides that functionality.
You can split channels into various profiles and organize your subscriptions feed that way.
In case you don't know where that feature is located, it's on the right top corner of the app.
![image](https://user-images.githubusercontent.com/41585298/116786329-8b5a6580-aa8d-11eb-856b-368ee7955af7.png)

**Comment by mrlen76** [comment]
While that would be considered a "workaround", it's definitely not an actual or even viable solution to the problem. I'd have to unsubscribe from those channels form my main profile and resubscribe to them on several other profiles (very messy). Then I'd have to sit there clicking through the profiles constantly to see if they have any new videos (defeating the purpose of having a subscription page) -- which I am definitely not going to do. If I'm going to do that I may as well just click their actual profiles (again, defeating the purpose of having a subscription page). I am quite disappointed that this was removed from featured requests because the amount of programming to add this functionality would be quite minimal really -- and the problem I have highlighted is a real and significant one. Who do I have to pay to get this added? And how much will it cost?
